src/Hosting/Hosting/test/Fakes/GenericWebHostBuilderWrapper.cs

```diff
@@ -16,7 +16,7 @@ public class GenericWebHostBuilderWrapper : IWebHostBuilder, ISupportsStartup, I
 
         internal GenericWebHostBuilderWrapper(HostBuilder hostBuilder)
         {
-            _builder = new GenericWebHostBuilder(hostBuilder);
+            _builder = new GenericWebHostBuilder(hostBuilder, new WebHostBuilderOptions());
```

Comment:
Note to self, if we do this for both generic host and web host then I think I can recombine the tests by adding a `Action<WebHostBuilderOptiosn>` to GenericWebHostBuilderWrapper.

Reply:
Are we doing this for both?

Reply:
No.